python/lsst/scarlet/lite/utils.py

```diff
@@ -0,0 +1,191 @@
+# This file is part of scarlet_lite.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import sys
+
+import numpy as np
+import numpy.typing as npt
+from scipy.special import erfc
+
+ScalarLike = bool | int | float | complex
+ScalarTypes = (bool, int, float, complex)
+
+
+def integrated_gaussian_psf(x: np.ndarray, sigma: float) -> np.ndarray:
+    """A Gaussian function evaluated at `x`
+
+    Parameters
+    ----------
+    x:
+        The coordinates to evaluate the integrated Gaussian.
+    sigma:
+        The standard deviation of the Gaussian.
+
+    Returns
+    -------
+    gaussian:
+        A Gaussian function integrated over `x`
+    """
+    sqrt2 = np.sqrt(2)
+    lhs = erfc((0.5 - x) / (sqrt2 * sigma))
+    rhs = erfc((2 * x + 1) / (2 * sqrt2 * sigma))
+    return np.sqrt(np.pi / 2) * sigma * (1 - lhs + 1 - rhs)
```

Comment:
Ah, no we actually want it the way that it's coded up. `erfc` _is_ normalized, and we are undoing that normalization by multiplying by the reciprocal of the normalization factor.

Reply:
We probably don't _have to_ do this.

My thinking is that we coded it up this way because we want to calculate the `integral(e^-t^2)` from `x` to `y` in this function, which is exactly what we return. Then it's up to the user to decide how to normalize that, instead of relying on a default normalization.